docs/layouts/shortcodes/generated/rocksdb_configurable_configuration.html

```diff
@@ -48,7 +48,7 @@
             <td><h5>state.backend.rocksdb.compaction.style</h5></td>
             <td style="word-wrap: break-word;">(none)</td>
             <td><p>Enum</p></td>
-            <td>The specified compaction style for DB. Candidate compaction style is LEVEL, FIFO or UNIVERSAL, and RocksDB choose 'LEVEL' as default style.<br /><br />Possible values:<ul><li>"LEVEL"</li><li>"UNIVERSAL"</li><li>"FIFO"</li></ul></td>
+            <td>The specified compaction style for DB. Candidate compaction style is LEVEL, FIFO or UNIVERSAL, and RocksDB choose 'LEVEL' as default style.<br /><br />Possible values:<ul><li>"LEVEL"</li><li>"UNIVERSAL"</li><li>"FIFO"</li><li>"NONE"</li></ul></td>
```

Comment:
If update RocksDB related configurations to latest, I think we should first update the description of `RocksDBConfigurableOptions#COMPACTION_STYLE`.
BTW, the web documentation should be updated via `mvn package -Dgenerate-config-docs -pl flink-docs -am -nsu -DskipTests` (refer to [flink-doc-README](https://github.com/apache/flink/blob/master/flink-docs/README.md))